src/editor/parts.js

```diff
@@ -15,6 +15,10 @@ limitations under the License.
 */
 
 import AutocompleteWrapperModel from "./autocomplete";
+import Avatar from "../Avatar";
+import MatrixClientPeg from "../MatrixClientPeg";
+
+const DPR = window.devicePixelRatio;
```

Comment:
DPR changes on page zoom... Is this really the value you want? If so, caching it here as a constant doesn't seem right to me...